src/views/appShell/AppLogIn/LoginPage.tsx

```diff
@@ -37,12 +37,23 @@ function Login() {
                     console.error('There was an error!', error);
                 })
                 .finally(() => {
-                    console.log('finally');
+                    setPassword('');
+                    setHasSubmitErrors(true);
```

Comment:
shouldn't this be in the catch error clause?

Reply:
set password should always be set to '' so this can stay in the finally clause. This way the plaintext password is not stored in memory on successful login. The `setHasSubmitErrors(true);` only needs to be called upon an error, so in the catch clause